frontend/src/components/auth/Login.tsx

```diff
@@ -1,6 +1,21 @@
 import React, { useContext, useState } from "react";
 import { Redirect, useHistory } from "react-router-dom";
 import { gql, useMutation } from "@apollo/client";
+import {
+  Flex,
+  Text,
+  Image,
+  FormControl,
+  FormLabel,
+  Link,
+  Box,
+  Button,
+  VStack,
+  Input,
+  Center,
+} from "@chakra-ui/react";
+import logo from "../logo.png";
+import background from "../signuppage.png";
```

Comment:
super nit: capitalize the name, and include "image" or "png" in the name? eg. `LogoBackgroundPng`